### PR TITLE
DOC: Usage about cal-model-memory

### PR DESCRIPTION
**Pull request by wxiwnd**

### CODE REVIEW

File: doc/source/models/model_memory.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  For example, Int4 for INT4 quantization.
+
+
+* --model-name {model_name}
+  
+  
+  * -n {model_name} *(Optional)*
+
+
+  Specify the model's name.
```
Comment (qinxuye) [inline doc/source/models/model_memory.rst:66]: Is model name optional?

Comment (wxiwnd) [reply doc/source/models/model_memory.rst:66]: Refer to 

https://github.com/xorbitsai/inference/pull/1460/commits/29e01f6d80b3cfbb8dcda2e5df9aec36bb154563#diff-f6f198b794a606bb897f2490939693ec0e6ccb7514aa81b2e253df6c177a24aeR91-R98

The model name is optional.

If provided, fetch model config from huggingface/modelscope

If not specified, use default model layer to estimate.

image

Comment (frostyplanet) [reply doc/source/models/model_memory.rst:66]: 中文建议改成：如果没有指定，就使用默认的layer参数粗略估计。
看到“粗略”二字用户肯定就知道不准确。

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**